benchmark/misc/string-creation.js

```diff
@@ -1,17 +0,0 @@
-'use strict';
-
-var common = require('../common.js');
-var bench = common.createBenchmark(main, {
-  millions: [100]
-});
-
-function main(conf) {
-  var n = +conf.millions * 1e6;
-  bench.start();
-  var s;
-  for (var i = 0; i < n; i++) {
-    s = '01234567890';
-    s[1] = 'a';
```

Comment:
Perhaps. With `use strict` it is definitely broken. Looking at the original commit ( https://github.com/nodejs/node/commit/12a169e8ff4036e160dbdc605eb9f15eae933b47 - 6 years ago) it seams like it was just a misunderstanding of how strings works. The commit appears to compare strings and buffers, which is not comparable in this case as strings are immutable.